# Service to set the parameters of a fiducial array

FiducialArrayInfo info
---
